README.md

```diff
@@ -27,19 +27,33 @@ In React, having a component that needs to have state is also a clue that it sho
 Here's an example of a React component whose state changes whenever its child element (a checkbox) is clicked.
 
 ```
+import React, { Component } from 'react' 
```

Comment:
If you're going to only import `Component` from React, then you need to update the syntax on line 32 to be `class myCheckBox extends Component {`